src/app/components_ngrx/assignee-selector/assignee-selector.component.html

```diff
@@ -21,12 +21,10 @@
   <ng-template #dropdownItem let-name="name"
     let-src="src + '&s=20'"
     let-stickontop="stickontop">
-    <div class="user-avatar margin-right-10 pull-left">
-      <img
-        class="user-assign-avatar pull-left"
-        [src]="src | almAvatarSize: 20"
-        onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'"/>
-    </div>
+    <user-avatar
+      [imgSrc]="src | almAvatarSize: 20"
+      onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'">
```

Comment:
No need to add this ``` onError ``` attributes here. Currently, it is same for all. and You already add ```onError``` in user-avatar.component.html.